src/fragments/lib/storage/flutter/list.mdx

```diff
@@ -1,4 +1,10 @@
-You can list all of the objects uploaded under a given prefix. This will list all public files (i.e. those with `guest` access level):
+You can list all of the objects uploaded under a given path.
+
+<BlockSwitcher>
+
+<Block name="Stable (Mobile)">
+
+This will list all public objects (i.e. those with `guest` access level):
```

Comment:
can we change objects to files? We should use that consistently across the Storage category as well.